### PR TITLE
Redux - store, actions, reducers

### PR DESCRIPTION
**Pull request by evans22j**
In this milestone, implemented the following;
- Add React-Redux (npm install react-redux).
- Structure your application files using a "feature folder"
- Configure the Redux Store

### CODE REVIEW

File: src/Redux/books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const ADDBOOK = 'bookstore/books/ADDBOOK';
+const REMOVEBOOK = 'bookstore/books/REMOVEBOOK';
+
+const booksReducer = (state = {}, action) => {
```
Comment (Kingstalux) [inline src/Redux/books.js]: - [x] Good job so far 👍 Kindly set the initial state in both the books and categories redux files to be an empty array 👍

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @evans22j, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966


## Highlights 👏 
✅ Good pull request description 🔥 
✅ Linters are passing 👍 
✅ Professional README file 💯 
✅ Correct Gitflow used 👏 


## Required Changes ♻️
_Check the comments under the review._

- [x] Also I cannot add or remove a book. Kindly fix the functionality of your app so that a user would be able to add and remove books using the redux store 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Redux/books.js (hunk 1 above)
Comment (evans22j) [reply src/Redux/books.js]: The add and remove book changes requested by the reviewer do not fall under this project. this project is a setup to add reducers and actions.

**Review by gemmen29 — APPROVED** [review]
Hi @evans22j, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**